mm2src/mm2_main/tests/docker_tests/tendermint_tests.rs

```diff
@@ -651,6 +651,32 @@ fn test_passive_coin_and_force_disable() {
     block_on(disable_coin_err(&mm, token, false));
 }
 
+#[test]
+fn test_tendermint_validators_rpc() {
+    let coins = json!([nucleus_testnet_conf()]);
+    let platform_coin = coins[0]["coin"].as_str().unwrap();
+
+    let conf = Mm2TestConf::seednode(TENDERMINT_TEST_SEED, &coins);
+    let mm = MarketMakerIt::start(conf.conf, conf.rpc_password, None).unwrap();
+
+    let activation_res = block_on(enable_tendermint(
+        &mm,
+        platform_coin,
+        &[],
+        NUCLEUS_TESTNET_RPC_URLS,
+        false,
+    ));
+    assert!(&activation_res.get("result").unwrap().get("address").is_some());
+
+    let validators_raw_response = block_on(tendermint_validators(&mm, platform_coin, "All", 10, 1));
+
+    assert_eq!(
+        validators_raw_response["result"]["validators"][0]["operator_address"],
+        "nucvaloper15d4sf4z6y0vk9dnum8yzkvr9c3wq4q897vefpu"
+    );
```

Comment:
Q: curious why this address we test against is constant. is this related to how we set up the docker data files for tendermint?

Reply:
Nodes run on the same state on each CI run. We expect `nucvaloper15d4sf4z6y0vk9dnum8yzkvr9c3wq4q897vefpu` to be a valid running validator on the chain.